orchestrator/src/main/java/com/scottlogic/deg/orchestrator/generate/GenerateCommandLine.java

```diff
@@ -105,51 +109,59 @@ private void checkForAlphaGenerationDataTypes() {
         description = "Determines the type of combination strategy used (${COMPLETION-CANDIDATES})")
     private CombinationStrategyType combinationType = MINIMAL;
 
+    @SuppressWarnings("FieldCanBeLocal")
     @CommandLine.Option(
         names = {"-n", "--max-rows"},
         description = "Defines the maximum number of rows that should be generated")
     private long maxRows = DEFAULT_MAX_ROWS;
 
+    @SuppressWarnings("FieldCanBeLocal")
```

Comment:
i don't get any warnings for these, does it need to be supressed? what's the warning about?
can it be supressed at class level, rather than on each field?

Reply:
warns me field can be converted to a local variable.  maybe you have added some suppressions to your local intellij project files?